stanford/playbooks/insights.yml

```diff
@@ -11,6 +11,7 @@
   become: True
   vars:
     CLUSTER_NAME: insights
+    COMMON_GIT_PATH: 'Stanford-Online'
```

Comment:
You may want to just set this value specifically for the `insights` role;
that's what needs it, right?

This variable, being in `COMMON_*`, is referenced by a number of other roles.
This is actually why we haven't overridden that value globally;
there are still a number of cases where we don't want to install from our fork.
Instead, we override it only when we need to.

Reply:
It is only set for the insights role. What am I missing?

Reply:
It's declared in the `vars` section for the entire _play_ `name: Deploy insights service`,
so I believe it would be enabled for everything inside this play, not just the role.

Look further down in the `roles` section, at `nginx`, for an example of setting a variable specifically for use w/in a role:

```yaml
roles:
  - role: insights
    COMMON_GIT_PATH: 'Stanford-Online'
```